types/humanize-ms/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for humanize-ms 1.2
+// Project: https://github.com/node-modules/humanize-ms
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function(input: string|number): number;
```

Comment:
the source library uses module.exports = not export default. The equivalent typescript construct is export = f, which is then imported with import HumanizeMs = require('humanize-ms')

Reply:
My mistake, I did the same thing here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33552. Wasn't aware of how to do default exports

Fixed 😄